docs/node.md

```diff
@@ -62,28 +62,30 @@ After creation, Kubernetes will check whether the node is valid or not.
 For example, if you try to create a node from the following content:
 ```json
 {
-  "id": "10.1.2.3",
-  "kind": "Minion",
-  "apiVersion": "v1beta1",
-  "resources": {
-    "capacity": {
-      "cpu": 1000,
-      "memory": 1073741824
-    },
-  },
-  "labels": {
-    "name": "my-first-k8s-node",
+  "kind": "Node",
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "10.240.79.157",
+    "labels": {
+      "name": "my-first-k8s-node"
+    }
   },
+  "status": {
+    "capacity": {
+      "cpu": "1k",
+      "memory": "1Gi"
+    }
+  }
 }
 ```
 
 Kubernetes will create a `Node` object internally (the representation), and
-validate the node by health checking based on the `id` field: we assume `id`
-can be resolved. If the node is valid, i.e. all necessary services are running,
-it is eligible to run a `Pod`; otherwise, it will be ignored for any cluster
-activity, until it becomes valid. Note that Kubernetes will keep invalid node
-unless explicitly deleted by client, and it will keep checking to see if it
-becomes valid.
+validate the node by health checking based on the `metadata.name` field: we
+assume `metadata.name` can be resolved. If the node is valid, i.e. all necessary
+services are running, it is eligible to run a `Pod`; otherwise, it will be
+ignored for any cluster activity, until it becomes valid. Note that Kubernetes
+will keep invalid node unless explicitly deleted by client, and it will keep
+checking to see if it becomes valid.
```

Comment:
The description is still true. The only small thing is that kubernetes does not need to keep checking -- node (kubelet) will post the status if it becomes ready.